documentation/guide-jpa-query.asciidoc

```diff
@@ -23,7 +23,59 @@ SELECT line FROM OrderLineEntity line WHERE line.order.id = :orderId
 ----
 
 == Dynamic Queries
-For dynamic queries we use http://www.querydsl.com/[QueryDSL]. It allows to implement queries in a powerful but readable and type-safe way (unlike Criteria API). If you already know JPQL you will quickly be able to read and write QueryDSL code. It feels like JPQL but implemented in Java instead of plain text.
+For dynamic queries, we use http://www.querydsl.com/[QueryDSL]. It allows to implement queries in a powerful but readable and type-safe way (unlike Criteria API). If you already know JPQL, you will quickly be able to read and write QueryDSL code. It feels like JPQL but implemented in Java instead of plain text.
```

Comment:
Can you specify that you show the usage of JPA Module from Querydsl and maybe that there are also more that is supported.